fixes for dragonflight and upping the limit to 35

CyberDracula opened the pull request:
Contains:
- fixes from [CurseForge comments](https://www.curseforge.com/wow/addons/cquestcounter?comment=42)
- upping the cap for quests from `25` to `35`
- Updating the `Interface` version to `100002`

CyberDracula committed:
fixes for dragonflight and upping the limit to 35

Hyphie24 commented:
I had stopped using CQuestCounter after finding SimpleQuestCounter, but I updated the code to work for Dragonflight. The font text can stay the same (that part hadn't been changed and wasn't "broken") but I moved the quest tracker to be more inline with the world map headers (to fix the scrolling issue)
Give it a try and tell me if you feel it is better (I tried to make it into the title box but it wasn't working as I had hoped and with addons like TomCats Tours & also the base wow campaign text) it was hard to see the number of quests from the addon

The only issue that I seem to see is if the zone you're in has 5 or more zone tabs (as the screenshot shows) it can "break" but it doesn't block you from any action in that frame

![image](https://user-images.githubusercontent.com/31455641/209567028-9b3237c7-2997-4884-bb47-fd486100bc03.png)

If everyone prefers to have it in the more natural spot of how it used to be I will push you're pull request through (since it does work)

https://github.com/Hyphie24/CQuestCounter/tree/Dragonflight-Update

CyberDracula commented:
> I had stopped using CQuestCounter after finding SimpleQuestCounter, but I updated the code to work for Dragonflight. The font text can stay the same (that part hadn't been changed and wasn't "broken") but I moved the quest tracker to be more inline with the world map headers (to fix the scrolling issue) Give it a try and tell me if you feel it is better (I tried to make it into the title box but it wasn't working as I had hoped and with addons like TomCats Tours & also the base wow campaign text) it was hard to see the number of quests from the addon
> 
> The only issue that I seem to see is if the zone you're in has 5 or more zone tabs (as the screenshot shows) it can "break" but it doesn't block you from any action in that frame
> 
> ![image](https://user-images.githubusercontent.com/31455641/209567028-9b3237c7-2997-4884-bb47-fd486100bc03.png)
> 
> If everyone prefers to have it in the more natural spot of how it used to be I will push you're pull request through (since it does work)
> 
> https://github.com/Hyphie24/CQuestCounter/tree/Dragonflight-Update

it looks good

MorningStarGG commented:
I changed the code to look better by tapping into a different frame:


Changing two lines:

Line #3
This: `<Frame name="CQCounterFrame" movable="false" parent="QuestScrollFrame">`
To this: `<Frame name="CQCounterFrame" movable="false" parent="WorldMapFrame">`


Line #10
And this: `<Anchor point="TOPLEFT" relativeTo="QuestScrollFrame" relativePoint="TOPLEFT" x="-120" y="-10">`
To this: `<Anchor point="TOPRIGHT" relativeTo="WorldMapFrame" relativePoint="TOPRIGHT" x="-120" y="-5">`


As you can see the style is similar to that of before DragonFlight and is not covering anything.

![Fixed](https://github.com/Hyphie24/CQuestCounter/assets/31865999/8d052026-a458-45ef-a31c-f19d7adae9e5)
